admin-site/src/Occasion.svelte

```diff
@@ -60,31 +82,33 @@
             getEventsAndStore();
         } else {
           response.text().then( errorMessage => {
-            console.log('Error on ' + state + ' request: ' + errorMessage)
+            console.log(`Error on ${state} request: ${errorMessage}`)
           })
         }
       }).catch( error => {
-        console.log("Catch error on " + state + ' request')
+        console.log(`Catch error on ${state} request`)
       })
     } catch (e) {
       console.log(e.message)
     }
   }
 
   function openOccasionButton() {
-    serverCallToOpenOrCloseOccasion("opened"); 
+    updateOccasionStateOnServer("opened"); 
   };
 
   function closeOccasionButton(){
-    serverCallToOpenOrCloseOccasion("closed");  
+    updateOccasionStateOnServer("closed");  
   };
 
 
   function deleteOccasion() {
+    //Delete Occasion is only available in the UI when an occasion is closed
```

Comment:
This is good but not quite the right amount of context. 

"Server will throw an error if we try to delete an open occasion, but we only show the Delete Occasion button when an occasion is closed, so we don't need to handle showing that error to users." (or something like that)

Reply:
Implemented